test/unit/auth/credential.spec.ts

```diff
@@ -328,6 +329,47 @@ describe('Credential', () => {
         });
       });
     });
+
+    it('should cache discovered project id', () => {
+      const expected = 'test-project-id';
```

Comment:
nit: Rename `expected` to `expectedProjectId` here and below.

Reply:
Done